### PR TITLE
adding basic uptime stats

### PR DESCRIPTION
**Pull request by matt-raffel-kiva**
Signed-off-by: Matt Raffel <mattr@kiva.org>

Following the same pattern as used in AGA, reporting etc....

### CODE REVIEW

File: src/app/dtos/service.report.dto.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { IsArray, IsString } from 'class-validator';
+
+/**
+ * Please note: this structure is expected to be used across all services
```
Comment (jeffk-kiva) [inline src/app/dtos/service.report.dto.ts:4]: If this object is going to be used in multiple projects, let's put it in a common library.

Comment (matt-raffel-kiva) [reply src/app/dtos/service.report.dto.ts:4]: I will make a separate PR once we have protocol-common in a good state.  I will not let it block getting this milestone complete however.

**Review by natesulat — APPROVED** [review]
+1 what Jeff said, but I don't think it needs to block this PR.

That said, do we have an Issue created for common-izing the DTO?

**Comment by matt-raffel-kiva** [comment]
> +1 what Jeff said, but I don't think it needs to block this PR.
> 
> That said, do we have an Issue created for common-izing the DTO?

as I said, I agree.  But until we finish unblocking protocol-common, I will not let that cause another feature to be blocked.  The issue is written up here: https://github.com/kiva/protocol-common/issues/75

**Pull request merged**